buildSrc/src/main/groovy/io.micronaut.build.internal.jms-examples.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'io.micronaut.build.internal.jms-base'
+    id 'io.micronaut.application'
+}
+
+micronaut {
+    version project.ext.micronautVersion
+    runtime 'netty'
+}
+
+dependencies {
+    testCompileOnly 'com.google.code.findbugs:jsr305' // for "warning: unknown enum constant When.MAYBE"
```

Comment:
Any reason you don't use the catalog for this one?

You could declare a library without version like this:

```toml
[libraries]
findbugs = { module = "com.google.code.findbugs:jsr305", version = "" }
```